velox/docs/configs.rst

```diff
@@ -129,6 +129,14 @@ Generic Configuration
      - 0
      - If it is not zero, specifies the time limit that a driver can continuously
        run on a thread before yield. If it is zero, then it no limit.
+   * - prefixsort_max_normalized_key_size
+     - integer
+     - 2147483647
```

Comment:
2GB is not a safe default value. Let's use something more reasonable. For example, 64 bytes. What is the max size we benchmarked with?